scripts/process/process.sparks.R

```diff
@@ -22,15 +20,16 @@ process.gage_sparks <- function(viz = as.viz('gage-sparks')){
   
   site.nos <- sites$site_no[which(sites$site_no %in% names(gage_data))]
 
-  sparks <- data.frame(points = sapply(site.nos, function(x) grab_spark(gage_data[[x]]$y), USE.NAMES = FALSE),
+  sparks <- data.frame(points = sapply(site.nos, function(x) grab_spark(gage_data[[x]]), USE.NAMES = FALSE),
                        site_no = site.nos, stringsAsFactors = FALSE) %>% 
     mutate(class = "sparkline", 
            id = sprintf("sparkline-%s", site_no), 
            style = "mask: url(#spark-opacity);",
            onmouseover=sprintf("setBold('nwis-%s');", site_no), 
            onmouseout=sprintf("setNormal('nwis-%s');hovertext(' ');", site_no),
-           onclick=sprintf("openNWIS('%s', evt);", site_no),
-           onmousemove=sprintf("hovertext('USGS %s',evt);", site_no))
+           onclick=sprintf("openNWIS('%s', evt);", site_no)) %>%
+    left_join(select(sites, site_no, station_nm), by="site_no") %>%
```

Comment:
A little worried that this join happens before some mouse events are declared but after others.

Reply:
Yeah, I bet this is where they got lost? This is how I did it over in the sites: https://github.com/USGS-VIZLAB/hurricane-maria/blob/master/scripts/process/process.classified_sites.R#L26 Not sure the same kind of thing will work over here?

Reply:
The spark opacity revel (slider) is no longer working too (the mask id)